### DISCUSSION TITLE
Does an official instance exist?

### DISCUSSION THREAD

**Discussion by beedell-roke** [searx/searx#3178] [Ideas]
https://github.com/searxng/searxng/discussions/2582#discussion-5454220

<blockQuote>

I want to use SearX due to its evident superiority to most other SEs in that it can aggregate results from multiple. However, https://searx.space/# shows quite a multitude of instances to choose from. Consequently, are any instances hosted (officially) by the developmental team?

If one doesn't exist, I believe that it should. I'm not the sole person to think that an official instance should exist, per https://www.reddit.com/r/privacytoolsIO/comments/h80oum/comment/g6wz1hr/?utm_source=share&utm_medium=web2x&context=3

> Would be neat if searx hosted an official server that was funded through donations or something.

Bitwarden generates quite a significant revenue stream for its project by providing an official instance, since it will always serve the latest version, and have the best uptime, fundamentally.

**Comment by unixfox** [comment]
No

**Reply by beedell-roke** [reply]
Thanks. The Q&A category really should be enabled.
